example/testapp/models.py

```diff
@@ -9,10 +9,13 @@ class Car(models.Model):
     def __unicode__(self):
         return self.plate
 
+class m2mprint:
+    def __unicode__(self):
```

Comment:
`__unicode__` is deprecated (in python 3 there's only unicode strings). Use `__str__` with [django.utils.encoding.python_2_unicode_compatible](https://docs.djangoproject.com/en/1.10/ref/utils/#django.utils.encoding.python_2_unicode_compatible).

Subclass a real django model.